node/service/src/lib.rs

```diff
@@ -94,6 +113,13 @@ impl sc_executor::NativeExecutionDispatch for MoonbeamExecutor {
 	}
 }
 
+#[cfg(feature = "moonbeam-native")]
+impl ExecutorT for MoonbeamExecutor {
+	fn first_block_number_compatible_with_ed25519_zebra() -> Option<u32> {
+		Some(2_000_000)
```

Comment:
>  this block numbers are meant to indicate the height at which a substrate version changing ed25519 library was introduced in each runtime

No, this block number indicates from which block the behavior of the host function should change, knowing that this only applies when synchronizing past blocks. Normally our runtimes are "compatible" with the new behavior since runtime 1100, but for safety we take a more recent block.